stable/artifactory-ha/templates/nginx-pvc.yaml

```diff
@@ -15,11 +15,11 @@ spec:
   resources:
     requests:
       storage: {{ .Values.nginx.persistence.size | quote }}
-{{- if .Values.nginx.persistence.storageClass }}
-{{- if (eq "-" .Values.nginx.persistence.storageClass) }}
+{{- if .Values.nginx.persistence.storageClassName }}
```

Comment:
Did you change it in the values.yaml?

Reply:
@danielezer  its already corrected in values.yaml